### PR TITLE
Fix 55 typos in 7 files

### PR DESCRIPTION
**Pull request by RoboSchmied**
Fix same typo 55 times

### CODE REVIEW

**Review by rminnich — DISMISSED** [review]

File: pkg/boot/image/image.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ var (
 	kernelImageSize = uint64(math.Pow(2, 24)) // 16MB, a guess value similar to that used in kexec-tools.
 
 	errBadMagic     = errors.New("bad header magic")
-	errBadEndianess = errors.New("invalid Image endianess, expected little")
+	errBadEndianness = errors.New("invalid Image endianness, expected little")
```
Comment (binjip978) [inline pkg/boot/image/image.go:26]: This file needs to be formatted with gofmt tool.

Comment (binjip978) [reply pkg/boot/image/image.go:26]: this command will do the trick:
```
find . -type f -name "*.go" ! -path "./vendor/*" -exec gofmt -w {} \;
```

**Review by RoboSchmied — COMMENTED** [review]
Thank you. I missed that.

File: pkg/boot/image/image.go (hunk 1 above)
Comment (RoboSchmied) [reply pkg/boot/image/image.go:26]: Thank you. I did it and committed the result.

**Review by binjip978 — APPROVED** [review]

**Pull request merged**